docs/Developers/Useful_Links.rst

```diff
@@ -0,0 +1,47 @@
+Useful Links
+============
+
+The purpose of this document is to gather links that coala
+developers usually use throughout their work and therefore
+make it easier for everybody to find and access that specific
+information.
+
+.. note::
+
```

Comment:
Right, don't make this bold. Too obtrusive. This will already be rendered in a box.